src/constants/reference_keys.rs

```diff
@@ -0,0 +1,48 @@
+//! Orchard reference keys (spending key and recipient address) used for reference notes.
+use crate::{
+    address::Address,
+    keys::{FullViewingKey, SpendingKey},
+};
+
+/// Raw bytes representation of the reference recipient address.
```

Comment:
Let's move the constant just above tests. It will be easier to understand how it is generated.

Reply:
Generating on the fly will require to perform expensive computation, thus, I prefer to store it
It is done similarly with Sinsemilla constants (see src/constants/sinsemilla.rs)
About the order in the file (constants, then functions to retrieve keys, then tests), I keep the same order than src/constants/sinsemilla.rs file.

Reply:
Done